helm/kubernetes-cluster-prep/templates/tests/_helm_test.bats.txt

```diff
@@ -74,4 +75,45 @@ text_color "$MAGENTA"
   assert_failure
 }
 {{- end }}
+
+@test "Conjur Account is valid" {
+  display_info "Validating account from the info endpoint for enterprise"
+
+  cmd=( curl -k --connect-timeout 5 "$conjurApplianceUrl"/info -o "$TEMP_INFO_FILE")
+  display_info "Running ${cmd[@]}"
+  run "${cmd[@]}"
+  run grep account.*"$conjurAccount" "$TEMP_INFO_FILE"
+
+  if [ "$status" -ne 0 ]; then
+    if (grep account "$TEMP_INFO_FILE"); then
+        status=0
+        display_error "Please check the configured Conjur Account.\n" \
+                      "It does not match with values from the info endpoint"
+        assert_failure
+    else
+        skip "test due to the info endpoint is not available for OSS"
+    fi
+  fi
+}
+
+@test "Conjur Authenticator is valid" {
+  display_info "Validating account from the info endpoint for enterprise"
+
+  cmd=( curl -k --connect-timeout 5 "$conjurApplianceUrl"/info -o "$TEMP_INFO_FILE")
+  display_info "Running ${cmd[@]}"
+  run "${cmd[@]}"
+  run grep name.*"$authnK8sAuthenticatorID" "$TEMP_INFO_FILE"
+
+  if [ "$status" -ne 0 ]; then
+    if (grep name "$TEMP_INFO_FILE"); then
+        status=0
```

Comment:
Same comment as above, we could drop the `status=0` and the `assert_failure`, and just use `assert_success` at the end of the test function, I think?

Reply:
Same reply as above